core/src/main/java/org/infinispan/container/offheap/BoundedOffHeapDataContainer.java

```diff
@@ -131,6 +131,28 @@ public void put(int segment, WrappedBytes key, WrappedBytes value, Metadata meta
                                   ice -> segments.contains(keyPartitioner.getSegment(ice.getKey())));
    }
 
+   @Override
+   public int sizeIncludingExpired(IntSet segments) {
```

Comment:
you changed some of the methods to long. would it be worth to put long instead of int in these new methods?

Reply:
Not at this point. The only method I changed to long was the internal value for size in the interceptor stack.